modules/ROOT/pages/aura-graphql/api-creation.adoc

```diff
@@ -0,0 +1,161 @@
+
+**Enable field suggestions** provides suggestions that hint towards GraphQL typos. 
+====
++
+. Type definitions
++
+This is where you describe the graph database in the AuraDB that the GraphQL API will be used with.
+The Type Definitions are the same as those used with Neo4j GraphQL Library with the exception that custom resolvers cannot be used.    
++
+If you already have data in the AuraDB, a quick way to obtain Type Definitions is to use the https://graphql-toolbox.neo4j.io[Neo4j GraphQL Toolbox].  This facility has the ability to connect to an AuraDB, automatically create Type Definitions and allow GraphQL operations.  
++
+Alternatively you can write your own Type Definition from first principles by following the guidance provided in xref:index.adoc[Neo4j GraphQL Library] documention
++
+If you are using GraphQL Federation then make sure to select **Enable GraphQL subgraph**.
++
+image::aura-graphql/create/type-definitions.png[]
++
+. Cross-Origin Resource Sharing (CORS) policy
++
+If a browser based application will be using this GraphQL API, add an entry in your Cross-Origin Resource Sharing (CORS) policy.
+CORS is a browser-based security mechanism that prevents web pages from accessing resources from a server on a different domain.
+Allow the domain that serves the browser application by adding it to the CORS policy.
+This includes development environments such as node.js, which serves content on your localhost or similar.
+This also holds for using web-based tooling for GraphQL APIs such as https://studio.apollographql.com/[Apollo Studio].
++
+This is not needed if a non-browser-based application is using the GraphQL API as CORS does not apply to those.
+For example, if you are trying out GraphQL operations using cURL. 
++
+[NOTE]
+====
+The domain entered in the CORS policy must be an exact match.
+For example, http://localhost is not the same as http://localhost:3000/.
+Wildcards are not supported. 
+====
++
+To add a CORS policy entry, enter the exact domain, including HTTP/S and any port number, in the **Origin box**.
+If you need multiple entries, select **Add allowed origin**.
++
+image::aura-graphql/create/cors-policy.png[]
++
+. Authentication providers
++
+All requests to the GraphQL API are authenticated and there are two options for the type of authentication: API key or JSON Web Key Set (JWKS).
+It is possible to use these in combination and have multiples of each.   
++
+After choosing the authentication provider from the dropdown list, enter a friendly name.
+This is all that is needed when using API key.
+The API key will be shown after the GraphQL API key has been created.
+If you are using JWKS, provide a friendly name and the URL from your identity provider that is used to verify JWKS tokens. 
++
+Add more more providers via **Add authentication provider**.
++
+An authentication provider can be removed by selecting the trash icon.
++
+image::aura-graphql/create/auth-providers.png[]
++
+[NOTE]
```

Comment:
Is it possible to make this a warning as it's an important note on the security risks of using API keys?